Assets/Scripts/Framework/Gameplay/PerformanceTracker/ScoreResults.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ProefExamen.Framework.Gameplay.PerformanceTracking
+{
+    /// <summary>
+    /// A scriptable object holding a list of results for levels.
+    /// </summary>
+    [CreateAssetMenu(fileName = "ScoreResults", menuName = "ScriptableObjects/ScoreResults")]
+    public class ScoreResults : ScriptableObject
+    {
+        /// <summary>
+        /// A list containing HighScores for levels.
+        /// </summary>
+        public List<PerformanceResult> highScores = new();
+    }
```

Comment:
I don't think its good idea to store high score data in a scriptable object, because we then have local changes everytime we beat the highscore. I think a simple and conventional way of storing game save data is using [Player Prefs](https://docs.unity3d.com/ScriptReference/PlayerPrefs.html).

Reply:
We'll have to make this a general discussion and talk about what we want to do.